CMakeLists.txt

```diff
@@ -168,9 +168,6 @@ else()
   message(STATUS "Building without build tag")
 endif()
 
-set(CMAKE_C_FLAGS_RELEASE "-DNDEBUG ${OPT_FLAGS_RELEASE}")
```

Comment:
You removed here also `${OPT_FLAGS_RELEASE}` if this was intended please remove the lines `158-161`